target/linux/lantiq/files-4.19/arch/mips/boot/dts/lantiq/grx390.dtsi

```diff
@@ -0,0 +1,322 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
```

Comment:
Please rename this file to xrx300.dtsi or xrx330.dtsi 
The name grx390 was never used in production

Reply:
renamed to xrx330